log4j-1.2-api/src/test/java/org/apache/log4j/layout/Log4j1SyslogLayoutTest.java

```diff
@@ -71,15 +72,18 @@ static Stream<Arguments> configurations() {
     public void testSimpleLayout(String expected, Facility facility, boolean header, boolean facilityPrinting) {
         final LogEvent logEvent = createLogEvent();
         StringLayout appenderLayout = Log4j1SyslogLayout.newBuilder()
+                .setConfiguration(new DefaultConfiguration())
```

Comment:
Nit-pick, though it shouldn't affect the test: all these layouts can use the same `Configuration` instance.